### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by Eyalon** [speedyHKjournalist/OpenMapleClient#4]
App dont work. After type host IP its goes Black screen.

**Comment by speedyHKjournalist** [comment]
hi, did you copy nx files to desired location and opened up a server?

**Comment by Eyalon** [comment]
Yeah i done everything like You write in instructions. Black screen goes up everytime after entering IP. Regardless that MapleServ APK is on or off.

**Comment by speedyHKjournalist** [comment]
https://www.youtube.com/watch?v=rqsi8sFt808&t=5s

You can have a look at this video, I started android maple server locally, then started client.

**Comment by Eyalon** [comment]
I think something is not right with APK. Not read wz files. When i delete everything and start APK without wz files, its act the same with black screen.
